### ISSUE TITLE
View location bug with Symfony 2.8.3

### ISSUE THREAD

**Issue by brechtdewinne** [symfony/symfony#17969]
When upgrading Symfony from 2.8.2 to 2.8.3 I have noticed that the following no longer works:

```php
return $this->container->get('templating')->renderResponse(
    '@BaseBundle/SubDir/Resources/views/view.html.twig', []
);
```
Directory structure: `/src/Base/SubDir/Resources/views/view.html.twig`

Symfony throws an exception saying that the view cannot be found inside `app/Resources/views`.

**Comment by iltar** [comment]
I'm not sure if that was ever supported as the default structure would've been loading templates from your view directory.

**Comment by brechtdewinne** [comment]
It have been using this long before 2.8 :)

**Comment by iltar** [comment]
That doesn't mean it's a supported notation though, I wasn't aware that this was possible in the first place. Supported notations are here: http://symfony.com/doc/current/book/templating.html#template-naming-and-locations

If this is intended to work, it should be fixed of course.

**Comment by brechtdewinne** [comment]
> Namespaced path support was introduced in 2.2.

Unless support for this has been removed and I missed that along the way.

**Comment by iltar** [comment]
You could add your own twig namespace: http://symfony.com/doc/current/cookbook/templating/namespaced_paths.html

**Comment by brechtdewinne** [comment]
And why would I want to that when it worked perfectly fine before?

This sounds more like a workaround for a bug then anything else...

**Comment by iltar** [comment]
I'm not saying you'd want to do that. I'm saying you could do that to get it to work in the newer version. If it's a bug, it can be fixed. If it was something that was possible but not intended to be, your solution will keep your system working. However, I recommend to stick to the documentation or best-practices regarding template locations.

**Comment by Jean85** [comment]
I think this is the same as #17777

**Comment by iltar** [comment]
In that case, @brechtdewinne can you try if this fix works for you? symfony/symfony#17894

**Comment by jakzal** [comment]
Indeed, this should be fixed by #17894.

The problem in my view is that people were relying on few unsupported cases, which obviously were not covered by tests.

**Comment by brechtdewinne** [comment]
@iltar Yes that fixes the issue!
CC: @Jean85 @jakzal
